packages/kolibri-common/strings/enhancedQuizManagementStrings.js

```diff
@@ -112,6 +90,13 @@ export const enhancedQuizManagementStrings = createTranslator('EnhancedQuizManag
   replaceAction: {
     message: 'Replace',
   },
+  noLearnersEnrolled: {
+    message: 'No learners enrolled in { className }',
+  },
+  noResourcesAvailable: {
+    message:
+      'There are no resources on your device yet. Ask an administrator to add resources to your device.',
```

Comment:
Don't we have these 2 strings already in some common strings file?

Reply:
cc @marcellamaki @rtibbles

Reply:
No

Reply:
Similar ones, but neither of these :)